core/traits/long_neck.py

```diff
@@ -0,0 +1,18 @@
+from core.constants import Signal, TraitCategory
+from core.trait_card import TraitCard
+from pydispatch import dispatcher
+
+
+class LongNeck(TraitCard):
+    def __init__(self):
+        super().__init__(type(self).__name__, tuple([TraitCategory.EATING]))
+        dispatcher.connect(self.handler_start_feeding_phase, Signal.START_FEEDING_PHASE)
+
+    def handler_start_feeding_phase(self):
+        # self._owner can be None, otherwise expect it to be a reference.
+        # The reference may resolve to None, so we check for that too.
```

Comment:
Am I right in thinking that self._owner would be None where the trait card is not yet played as a trait card on a species?

Reply:
Yeah, that's right -- means that we can do tests etc on a TraitCard() even if it's not owned by a species.

Reply:
Roger!